### PR TITLE
Fix a few notebook promises

### PR DESCRIPTION
**Pull request by aaomidi**

### CODE REVIEW

File: src/sql/workbench/contrib/jobManagement/browser/notebooksView.component.ts [hunk 1]
```diff
@@ -407,14 +409,14 @@ export class NotebooksViewComponent extends JobManagementView implements OnInit,
 					break;
 				}
 			}
-			this.openLastNRun(targetNotebook, barId, 5);
+			this.openLastNRun(targetNotebook, barId, 5).catch(e => this._logService.error(e));
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/jobManagement/browser/notebooksView.component.ts]: Any reason not to use onUnexpectedError instead?

File: src/sql/workbench/api/browser/mainThreadNotebook.ts [hunk 2]
```diff
@@ -235,7 +235,9 @@ class SessionManagerWrapper implements azdata.nb.SessionManager {
 
 	private async doStartNew(options: azdata.nb.ISessionOptions): Promise<azdata.nb.ISession> {
 		let sessionDetails = await this._proxy.ext.$startNewSession(this.managerHandle, options);
-		return new SessionWrapper(this._proxy, sessionDetails);
+		const sessionManager = new SessionWrapper(this._proxy, sessionDetails);
+		await sessionManager.initialize();
```
Comment (Charles-Gagnon) [inline src/sql/workbench/api/browser/mainThreadNotebook.ts]: Where was initialize being called before this? Is doing it here going to break anything?

File: src/sql/workbench/api/browser/mainThreadNotebook.ts [hunk 3]
```diff
@@ -258,8 +260,13 @@ class SessionManagerWrapper implements azdata.nb.SessionManager {
 class SessionWrapper implements azdata.nb.ISession {
 	private _kernel: KernelWrapper;
 	constructor(private _proxy: Proxies, private sessionDetails: INotebookSessionDetails) {
```
Comment (chlafreniere) [inline src/sql/workbench/api/browser/mainThreadNotebook.ts]: while you're here, if you could put an _ before sessionDetails that'd be great 😄

**Comment by aaomidi** [comment]
@Charles-Gagnon I'm implementing the tslint rule we had in eslint now - so just fixing up the problems before I get them in.

**Comment by chlafreniere** [comment]
> @Charles-Gagnon I'm implementing the tslint rule we had in eslint now - so just fixing up the problems before I get them in.

Did you get that backwards, or am I very confused 😄

File: src/sql/workbench/contrib/jobManagement/browser/notebooksView.component.ts (hunk 1 above)
Comment (aaomidi) [reply src/sql/workbench/contrib/jobManagement/browser/notebooksView.component.ts]: From where?

Comment (Charles-Gagnon) [reply src/sql/workbench/contrib/jobManagement/browser/notebooksView.component.ts]: https://github.com/microsoft/azuredatastudio/blob/master/src/vs/base/common/errors.ts#L77

**Comment by aaomidi** [comment]
Sorry I re-read and it was confusing.

TSLint is deprecated, so we've moved off of it. ESLint now has a typescript plugin with all the rules.

File: src/sql/workbench/contrib/jobManagement/browser/notebooksView.component.ts (hunk 1 above)
Comment (aaomidi) [reply src/sql/workbench/contrib/jobManagement/browser/notebooksView.component.ts]: Didn't know this was a thing, I guess that's a good idea

**Pull request merged**